static/app/views/dashboardsV2/widget/issueWidget/fields.tsx

```diff
@@ -0,0 +1,38 @@
+export type ColumnType =
+  | 'boolean'
+  | 'date'
+  | 'duration'
+  | 'integer'
+  | 'number'
+  | 'percentage'
+  | 'string';
+
+enum FieldKey {
+  ASSIGNEE = 'assignee',
+  TITLE = 'title',
+  ISSUE = 'issue',
+  LEVEL = 'level',
+  STATUS = 'status',
+  PLATFORM = 'platform',
+  PERMALINK = 'permalink',
+  IS_BOOKMARKED = 'isBookmarked',
+  IS_PUBLIC = 'isPublic',
+  IS_SUBSCRIBED = 'isSubscribed',
+  IS_HANDLED = 'isHandled',
+  CULPRIT = 'culprit',
+}
+
+export const ISSUE_FIELDS: Readonly<Record<FieldKey, ColumnType>> = {
+  [FieldKey.ASSIGNEE]: 'string',
+  [FieldKey.TITLE]: 'string',
+  [FieldKey.ISSUE]: 'string',
+  [FieldKey.LEVEL]: 'string',
+  [FieldKey.STATUS]: 'string',
+  [FieldKey.PLATFORM]: 'string',
+  [FieldKey.PERMALINK]: 'string',
+  [FieldKey.IS_BOOKMARKED]: 'boolean',
+  [FieldKey.IS_PUBLIC]: 'boolean',
```

Comment:
Not sure what this is, but is this something we expose to customers in the issue page? Should we remove it for now?

Reply:
Removed 👍